node-js-basics-101/node_modules/uws/uws.js

```diff
@@ -0,0 +1,563 @@
+'use strict';
+
+const http = require('http');
+const EventEmitter = require('events');
+const EE_ERROR = 'Registering more than one listener to a WebSocket is not supported.';
+const DEFAULT_PAYLOAD_LIMIT = 16777216;
+
+function noop() {}
+
+function abortConnection(socket, code, name) {
+    socket.end('HTTP/1.1 ' + code + ' ' + name + '\r\n\r\n');
+}
+
+function emitConnection(ws) {
+    this.emit('connection', ws);
+}
+
+function onServerMessage(message, webSocket) {
+    webSocket.internalOnMessage(message);
+}
+
+const native = (() => {
+    try {
+        try {
+            return process.binding('uws_builtin');
+        } catch (e) {
+            return require(`./uws_${process.platform}_${process.versions.modules}`);
```

Comment:
__[jshint]__
* 'template literal syntax' is only available in ES6 (use 'esversion: 6').

<!-- Comment by [SideCI](https://sideci.com) -->